cocos/core/load-pipeline/uuid-loader.ts

```diff
@@ -174,6 +175,16 @@ function loadDepends (pipeline, item, asset, depends, callback) {
                 }
             }
         }
+
+        if (asset instanceof legacyCC.SceneAsset && asset.scene) {
+            const dependAssets = dependKeys.concat();
+            for (const dependUuid of dependKeys) {
```

Comment:
Please use standard for instead of for of

Reply:
OK, I used a map instead of an array